### PR TITLE
RELEASING: documnent pushing packages to CMD

### PR DESCRIPTION
**Pull request by gwarf**
Document pushing packages to CMD.

### CODE REVIEW

**Comment by gwarf** [comment]
@enol, could you please review the update?

File: RELEASING.md [hunk 1]
```diff
@@ -71,3 +71,18 @@ git pull-request
...
+[EGI Software provisioning process](https://wiki.egi.eu/wiki/EGI_Software_Provisioning).
+
+* Submitting a [GGUS ticket](https://ggus.eu/?mode=ticket_submit) to the
+  `EGI Software provisioning support` Support Unit
+  * Ticket category: release
+  * Type of issue: Middleware
+  * Assign to support unit: `EGI Software Provisioning Support`
+  * Version number
+  * Link to release notes (GitHub and AppDB release pages)
+  * List of packages to release in UMD
```
Comment (enolfc) [inline RELEASING.md]: nitpick s/UMD/CMD/

Comment (gwarf) [reply RELEASING.md]: Done, also added a link to CMD.

**Comment by gwarf** [comment]
@enolfc, ready for you!

**Pull request merged**